assets/css/style.css

```
@@ -112,6 +112,12 @@ header svg {
 #directml-logo {
   height: 40px;
   width: 110px;
+  display: grid;
+  align-items: center;
+  font-size: 27px;
+  font-family: 'Segoe UI', SegoeUI, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

```suggestion
  font-family: "Segoe UI", SegoeUI, "Helvetica Neue", Helvetica, Arial, sans-serif;
```
or
```suggestion
  font-family: 'Segoe UI', SegoeUI, 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Technically either are allowed in CSS (but I generally lean to double quotes because some rare fonts actually have apostrophes in their name).

---

Fixed to double quotes, sorry about that.